target/scripts/startup/setup-stack.sh

```diff
@@ -95,19 +102,45 @@ function _setup_rspamd
 
   if [[ ${ENABLE_AMAVIS} -eq 1 ]] || [[ ${ENABLE_SPAMASSASSIN} -eq 1 ]]
   then
-    _shutdown 'You cannot run Amavis/SpamAssassin and Rspamd at the same time'
+    _log 'warn' 'Running rspamd at the same time as Amavis or SpamAssassin is discouraged'
   fi
 
   if [[ ${ENABLE_CLAMAV} -eq 1 ]]
   then
     _log 'debug' 'Rspamd will use ClamAV'
-    mv /etc/rspamd/local.d/disabled/antivirus.conf /etc/rspamd/local.d/antivirus.conf
+    sedfile -i -E 's|^(enabled).*|\1 = true;|g' /etc/rspamd/local.d/antivirus.conf
+    # RSpamd uses ClamAV's UNIX socket, and to be able to read it, it must be in the same group
+    usermod -a -G clamav _rspamd
```

Comment:
Just wondering, if that is the correct user? What up with the underscore?

Reply:
> What up with the underscore?

:man_shrugging: it's a user created by the installed package. It is correct and can be found in `/etc/passwd`.

Reply:
Ack, I didn't use rspamd yet and couldn't find that the user was created manually in one of the setup scripts.

Reply:
For reference in [this comment](https://github.com/docker-mailserver/docker-mailserver/pull/3011#issuecomment-1399120252) on another PR, I have a collapsed section for a `setup-users.sh` script that details `/etc/passwd`, `/etc/group` and `/etc/shadow` with upstream links to each packages specific source lines for users and groups added. That includes the `_rspamd`.

Reply:
This is the correct user, as @polarathene pointed put :)